virt/release_notes/virt-4-14-release-notes.adoc

```diff
@@ -142,6 +145,9 @@ Deprecated features are included in the current release and supported. However,
 //CNV-26426 [DOCS] Release note: Deprecate TTO
 * The `tekton-tasks-operator` is deprecated and Tekton tasks and example pipelines are now deployed by the `ssp-operator`.
 
+//CNV-21651
+* The `copy-template`, `modify-template`, and `create-vm-from-template` tasks are deprecated.
```

Comment:
modify-vm-template *
@ksimon1 Hi Karel, we are marking tasks as deprecated but there is no decision to deprecate template . For now i see only 2012 templates are mark as deprecated?

Reply:
The templates will be sooner or lated deprecated. We agreed, that we will keep template based tasks marked as deprecated

Reply:
@danielclowers then just one comment please rename modify-template to modify-vm-template